lib/tasks/load_tests.rake

```diff
@@ -0,0 +1,13 @@
+require "benchmark"
+require "load_tester"
+
+namespace :load_tests do
+  desc "Run a load test of the DeliveryRequestWorker by triggering 83,333 requests"
+  task :delivery_request_worker, [] => :environment do |_t, _args|
+    results = Benchmark.measure do
+      LoadTester.new.trigger_delivery_request_workers(83333)
```

Comment:
I'd wonder if this would be slightly more idiomatic as a class method rather than doing new then calling it.

Also missed a great chance to use `83_333`